src/shell.ts

```diff
@@ -50,7 +50,7 @@ export interface ShellResult {
 export type ShellHandler = (code: number, stdout: string, stderr: string) => void;
 
 function isWindows(): boolean {
-    return (process.platform === WINDOWS);
+    return (process.platform === WINDOWS) && !getUseWsl();
```

Comment:
Think you need to update the `platform()` function too, which also examines `process.platform`.  `platform()` is used for installing tools so need to consider whether these get correctly installed into WSL in this situation (e.g. we download and untar to the file system directly, and currently assume it's the host file system).

Reply:
Install now works correctly.